lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -3447,6 +3450,43 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
                 , expectField (#assets . #total . #getApiT)
                     (`shouldBe` tokens')
                 ]
+
+    it "TRANS_NEW_CREATE_11a - Get policy id - incorrect template \
+        \" $ \ctx -> runResourceT $ do
+        wa <- fixtureWallet ctx
+        let payload = Json [json|{
+                "policy_script_template":
+                    { "all":
+                       [ { "active_from": 120 }
+                       ]
+                    },
+                "asset_name": "ab12"
```

Comment:
Hm, I'm quite sure that `asset_name` is not needed for getting a policy id... I believe that we need only `policy_script_template`.

Reply:
indeed! I will revert. script template and policy public key are needed here! thanks